log.py

```diff
@@ -162,27 +269,30 @@ def from_configuration(cls, _db, testing=False):
             loggly = ExternalIntegration.lookup(
                 _db, ExternalIntegration.LOGGLY, goal
             )
+
```

Comment:
This code works, but you haven't accomplished the refactoring I was hoping for. I believe this entire method can be replaced with something like this:

```
handlers = []
handlers.append(Syslog.from_configuration(_db))
loggly = Loggly.from_configuration(_db)
if loggly:
    handlers.append(loggly)
cls.replace_handlers(_db, handlers)
```

Most of the logic currently in this method would go into `Syslog.from_configuration`; most of the rest would go into `LogConfiguration.replace_handlers`. You've already got most of the Loggly code in `Loggly.from_configuration`, but this would include the part that finds the ExternalIntegration for a Loggly configuration and pulls configuration settings from the database.

Probably you would end up getting the sitewide settings multiple times -- once for the syslog, once for the Loggly configuration, and a third time to set the log levels -- but that's fine because this code is only called once, on startup -- it's more important that the code be modular and easy to read. Right now it's kind of a mess (which is my fault as I wrote the original code). You've made it work with the new system, but it still should be refactored so that all the code pertaining to the syslogger goes into `Syslogger`.